Cargo.toml

```
@@ -49,6 +49,8 @@ opentelemetry-otlp = { version = "0.15.0", features = [
     "tls-roots",
 ], optional = true }
 opentelemetry-system-metrics = { version = "0.1.8", optional = true }
+tailcall-http-cache = {path = "tailcall-http-cache", optional = true }
```

```suggestion
tailcall-http-cache = {path = "tailcall-http-cache" }
```

Why is this optional?

---

as per comments in cargo.toml file, `http-cache-reqwest` has some conflicts with wasm and it's conditionally enabled for cli module and not keeping it optional was causing this integration to fail initially.